specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -3261,6 +3261,98 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/registries/{registryName}/models/{modelName}/versions/{version}/package": {
```

Comment:
>package [](http://example.com/codeflow?start=180&length=7)

Although package is a verb, it is very commonly a noun. Consider using a more action-y sounding name like buildPackage, makePackage, createPackate, or similar to more strongly indicate performing an action on a resource.